src/worker/deduplicate-installations.ts

```diff
@@ -0,0 +1,32 @@
+import { queues } from "./main";
+import { getLogger } from "../config/logger";
+
+const logger = getLogger("deduplicate-installations");
+
+export const deduplicateInstallations = async() : Promise<number> => {
+	// This remove all jobs from the queue. This way,
+	// the whole queue will be drained and all jobs will be readded.
+	const jobs = await queues.installation.getJobs(["active", "delayed", "waiting", "paused"]);
+	const foundInstallationIds = new Set<number>();
+	const duplicateJobs = [];
+
+	// collecting duplicate jobs per installation
+	for (const job of jobs) {
+		if (!job) {
+			continue;
+		}
```

Comment:
added a null check, because if `getJobs` doesn't find any jobs, it seems to return an array with a `null` in it :/